roles/kubernetes/master/templates/kubeadm-config.v1beta1.yaml.j2

```diff
@@ -38,7 +38,7 @@ networking:
   serviceSubnet: {{ kube_service_addresses }}
   podSubnet: {{ kube_pods_subnet }}
 kubernetesVersion: {{ kube_version }}
-{% if groups['kube-master'] | length > 1 and kubeadm_config_api_fqdn is defined %}
+{% if groups['kube-master'] | length >= 1 and kubeadm_config_api_fqdn is defined %}
```

Comment:
v1alpha2 v1alpha2 and v1alpha3  need to change too.

Reply:
Done